tests/config/pytest_modules.yml

```
@@ -2134,6 +2134,10 @@ plasmidfinder:
 plasmidid:
   - modules/nf-core/plasmidid/**
   - tests/modules/nf-core/plasmidid/**
+ 
+platypus:
+  - modules/platypus/**
+  - tests/modules/platypus/**
```

```suggestion
  - modules/nf-core/platypus/**
  - tests/modules/nf-core/platypus/**
```

You were still using the old file paths in the tests (I overlooked it sorry :D)